common/buildcraft/energy/BCEnergyConfig.java

```diff
@@ -32,6 +32,17 @@
     public static boolean enableOilGeneration;
     public static double oilWellGenerationRate;
     public static boolean enableOilSpouts;
+    public static boolean enableOilBurn;
+
+    public static int smallSpoutMinHeight;
+    public static int smallSpoutMaxHeight;
+    public static int largeSpoutMinHeight;
+    public static int largeSpoutMaxHeight;
+
+    public static Double smallOilGenProb;
+    public static Double mediumOilGenProb;
+    public static Double largeOilGenProb;
```

Comment:
Did you mean to make these objects? (Rather than primitives - `double` vs `Double`)